docs/csharp/tour-of-csharp/index.yml

```diff
@@ -0,0 +1,146 @@
+          - text: Display command-line
+            url: ../fundamentals/tutorials/how-to-display-command-line-arguments.md
+          - text: Intro to classes
+            url: ../fundamentals/tutorials/classes.md
+          - text: Object oriented C#
+            url: ../fundamentals/tutorials/oop.md
+          - text: Converting types
+            url: ../fundamentals/tutorials/safely-cast-using-pattern-matching-is-and-as-operators.md
+          - text: Pattern matching
+            url: ../fundamentals/tutorials/pattern-matching.md
+          - text: Use LINQ to query data
+            url: ../tutorials/working-with-linq.md
+
+  - title: "Key concepts"
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: C# language strategy
+            url: ../tour-of-csharp/strategy.md
+          - text: "Programming concepts"
+            url: ../programming-guide/concepts/index.md
+      - linkListType: concept
+        links:
+          - text: "Language Integrated Query (LINQ)"
+            url: ../linq/index.md
+          - text: "Asynchronous programming"
+            url: ../programming-guide/concepts/async/index.md
+
+  - title: "Advanced concepts"
+    linkLists:
+      - linkListType: reference
+        links:
+          - text: "Reflection and attributes"
+            url: ../advanced-topics/reflection-and-attributes/index.md
+          - text: "Expression trees"
+            url: ../advanced-topics/expression-trees/index.md
+          - text: "Native interoperability"
+            url: ../advanced-topics/interop/index.md
+          - text: "Performance engineering"
+            url: ../advanced-topics/performance/index.md
+          - text: ".NET Compiler Platform SDK"
+            url: ../roslyn-sdk/index.md
+
+  - title: "What's new"
+    linkLists:
+      - linkListType: whats-new
+        links:
+          - text: "What's new in C# 13"
+            url: ../whats-new/csharp-13.md
+          - text: "What's new in C# 12"
+            url: ../whats-new/csharp-12.md
+          - text: "What's new in C# 11"
+            url: ../whats-new/csharp-11.md
+          - text: "What's new in C# 10"
+            url: ../whats-new/csharp-10.md
+      - linkListType: tutorial
+        links:
+          - text: Explore record types
+            url: ../whats-new/tutorials/records.md
+          - text: Explore top level statements
+            url: ../whats-new/tutorials/top-level-statements.md
+          - text: Explore new patterns
+            url: ../whats-new/tutorials/patterns-objects.md
+          - text: Write a custom string interpolation handler
+            url: ../whats-new/tutorials/interpolated-string-handler.md
+      - linkListType: reference
+        links:
+          - text: Breaking changes in the C# compiler
+            url: ../whats-new/breaking-changes.md
+          - text: Version compatibility
+            url: ../whats-new/version-update-considerations.md
+            
+  - title: "Stay in touch"
```

Comment:
I suggest putting this card above the What's new so the page layout is more "even".

Reply:
<img width="496" alt="image" src="https://github.com/dotnet/docs/assets/24882762/14818a08-2cdc-4761-83de-95c428137e34">